msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -1109,12 +1109,13 @@ The `extensions` property specifies Outlook Add-ins within an app manifest and s
 |`alternates`| Array | 10 | | Specifies the relationship to alternate existing Microsoft 365 solutions. It's used to hide or prioritize add-ins from the same publisher with overlapping functionality. |
 |`audienceClaimUrl`| String | 2048 characters | | Specifies the URL for your extension and is used to validate Exchange user identity tokens. For more information, see [inside the Exchange identity token](/office/dev/add-ins/outlook/inside-the-identity-token)|
 |`appDeeplinks`| Array | | | Do not use. For Microsoft internal use only. |
+|`getStartedMessages`| Array | 3 | | Provides information about an Office Add-in that appears in the callout that opens in Office when an Office Add-in is installed. |
```

Comment:
Does "opens in Office" here refer to the Harmony (office.com / microsoft365.com) client? Or is "Office" here the generic term for one of the WXPO applications?

Reply:
Hi @Rick-Kirkham,
Request you to make the required changes. Thank you.